modules/oe_theme_helper/oe_theme_helper.post_update.php

```diff
@@ -41,3 +42,19 @@ function oe_theme_helper_post_update_use_retina_image_styles(array &$sandbox): v
     }
   }
 }
+
+/**
+ * Change the region of the search block.
+ */
+function oe_theme_helper_post_update_change_region_of_search_block() {
```

Comment:
Post update methods need a numeric identifier first to ensure they are applied in the correct order.

Reply:
trueeeeeeeeeeee